macros/generate_insert_statements_post_hook.sql

```diff
@@ -0,0 +1,21 @@
+{% macro generate_insert_statements_post_hook(relation, resource_type='nodes', relationships=False, batch_size=var('insert_batch_size', 500 if target.type == 'bigquery' else 10000)) %}
+  {%- set values = get_resource_values(resource_type, relationships) -%}
+  {%- set values_length = values | length -%}
+  {%- set loop_count = (values_length / batch_size) | round(0, 'ceil') | int -%}
+  
+    {%- for loop_number in range(loop_count) -%}
+        {%- set lower_bound = loop.index0 * batch_size -%}
+        {%- set upper_bound = loop.index * batch_size -%}
+        {%- set values_subset = values[lower_bound : upper_bound] %}
+        {%- set values_list_of_strings = [] -%}
+        {%- for indiv_values in values_subset %}
+            {%- do values_list_of_strings.append( indiv_values | join(", \n")) -%}
+        {%- endfor -%}
+        {%- set values_string = '(' ~ values_list_of_strings | join("), \n\n(") ~ ')' %}
```

Comment:
do we need to make this a separate set variable, or can we just add this logic into the `set insert_statement`?

Reply:
not any need besides reducing the number of `~`s in a single line 😆

Reply:
Ok cool! I'm fine with it then, just checking